Cannot find module 'tslib'

I'm trying to use graphile-migrate on Cloud Build with npx and it couldn't find tslib. I confirmed this by installing graphile-migrate into an empty package.json

```
$ graphile-migrate migrate
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'tslib'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/workspace/packages/web-app/database/node_modules/graphile-migrate/dist/index.js:3:17)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

--- a/package.json
+++ b/package.json
@@ -44,7 +44,8 @@
     "chokidar": "^3.1.1",
     "pg": ">=6.5 <8",
     "pg-connection-string": "^2.1.0",
-    "pg-minify": "^1.5.1"
+    "pg-minify": "^1.5.1",
+    "tslib": "^1.10.0"
   },
   "devDependencies": {
     "@types/jest": "^24.0.18",

